netmanager/src/views/components/Copy/Copyable.js

```diff
@@ -55,7 +56,9 @@ const Copyable = ({ value, className, width, format }) => {
       {copied ? (
         <CopySuccessIcon />
       ) : (
-        <CopyIcon onClick={onClick} className={className} />
+        <Button disabled={value === "-"} onClick={onClick}>
```

Comment:
please shed some light on this logic. Thanks @Codebmk

Reply:
When you copy an empty key, it returns the value of '-'. So I am disabling the copy button if the value is '-'

Reply:
Alright, makes sense to me then! Thanks @Codebmk